core/pom.xml

```diff
@@ -174,6 +174,10 @@
       <groupId>jakarta.servlet</groupId>
       <artifactId>jakarta.servlet-api</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I think `javax.activation:activation` and `jakarta.activation` should be consolidated into one. Should we use `jakarta.activation` with a specific implementation here, rather than just the API?

also cc @pan3793

Reply:
According to https://eclipse.dev/jetty/, Jetty 11 is compatible with JakartaEE 9, ideally, we should align all related deps with JakartaEE 9.